tests/unit/test_artifactory_path.py

```diff
@@ -667,6 +667,20 @@ def test_path_in_repo(self):
         b = P("http://b/artifactory/reponame/folder/path.xml")
         self.assertEqual(b.path_in_repo, "/folder/path.xml")
 
+    def test_joinpath_repo(self):
+        """
+        https://github.com/devopshq/artifactory/issues/239
```

Comment:
A link without short description doesn't give an information and a developer will have to go to browser.
Short note can be like 
```
We ignore lack of or additional / symbol between base url and reponame in `jointpath` operation

There was an issue about it: https://github.com/devopshq/artifactory/issues/239
```

Reply:
Good point.
As there are some existed test case containing a single URL without description, I mistakenly thought it was a special style we followed.

I will add additional information in future. Again, nice project.

> A link without short description doesn't give an information and a developer will have to go to browser.
> Short note can be like
> 
> ```
> We ignore lack of or additional / symbol between base url and reponame in `jointpath` operation
> 
> There was an issue about it: https://github.com/devopshq/artifactory/issues/239
> ```

Good point.
As there are some existed test case containing a single URL without description, I mistakenly thought it was a special style we followed.

I will add additional information in future. Again, nice project.